ccdproc/image_collection.py

```diff
@@ -121,14 +124,14 @@ def summary(self):
         If an explicit list of keywords was supplied in setting up the
         collection then the order of the columns is the order of the
         keywords.
-        """,
+        """
         return self._summary_info
 
     @property
     def summary_info(self):
         """
-        Deprecated -- use summary instead -- astropy.table.Table of values of
-        FITS keywords for files in the collection.
+        Deprecated -- use `summary` instead -- `~astropy.table.Table` of values
```

Comment:
Yes it does. Inside a class linking to attributes works without specifying the class. I hope that's something sphinx/readthedocs do by default and there is no special setting on my computer.